app/lib/three_scale/redis_config.rb

```diff
@@ -4,16 +4,14 @@ module ThreeScale
   class RedisConfig
     def initialize(redis_config = {})
       raw_config = (redis_config || {}).deep_symbolize_keys
-      sentinels = raw_config.delete(:sentinels).presence
       raw_config.delete_if { |key, value| value.blank? }
```

Comment:
IIRC the convention is
```suggestion
      raw_config.delete_if { |_key, value| value.blank? }
```